configs/hugetlb.go

```diff
@@ -0,0 +1,9 @@
+package configs
+
+type Hugetlb struct {
```

Comment:
I think the name `Hugetlb` makes you confused.
We already have HugetlbGroup in cgroup/fs/hugetlb.go
I will change the name of struct and file